Modules/Synchronization/src/Synchronization.Infrastructure/Persistence/Database/SynchronizationDbContext.cs

```diff
@@ -94,24 +94,20 @@ public async Task<DbPaginationResult<DatawalletModification>> GetDatawalletModif
         return datawallet;
     }
 
-    public async Task<ExternalEvent> CreateExternalEvent(IdentityAddress owner, ExternalEventType type, object payload)
+    public async Task CreateExternalEvent(ExternalEvent externalEvent)
     {
-        ExternalEvent? externalEvent = null;
-
         await RunInTransaction(async () =>
         {
-            if (externalEvent != null)
-                // if the transaction is retried, the old event has to be removed from the DbSet, because a new one with a new index is added
```

Comment:
Even with a single statement, surrounding that statement with curly braces enhances readability and even debugging

Reply:
You commented on removed code. :) The new code doesn't contain this problem anymore.